src/profile-logic/profile-data.js

```diff
@@ -1332,34 +1332,53 @@ export function getFriendlyThreadName(
   thread: Thread
 ): string {
   let label;
+
   switch (thread.name) {
-    case 'GeckoMain':
-      switch (thread.processType) {
-        case 'default':
-          label = 'Parent Process';
-          break;
-        case 'gpu':
-          label = 'GPU Process';
-          break;
-        case 'tab': {
-          const contentThreads = threads.filter(thread => {
-            return thread.name === 'GeckoMain' && thread.processType === 'tab';
-          });
-          if (contentThreads.length > 1) {
-            const index = 1 + contentThreads.indexOf(thread);
-            label = `Content Process (${index}/${contentThreads.length})`;
-          } else {
-            label = 'Content Process';
+    case 'GeckoMain': {
+      if (thread.hasOwnProperty('processName') && thread.processName) {
+        label = thread.processName;
+        const homonymThreads = threads.filter(thread => {
+          return (
+            thread.name === 'GeckoMain' &&
+            thread.hasOwnProperty('processName') &&
```

Comment:
I believe this `hasOwnProperty` check is redundant, as the next line `thread.processName === label` will be enough to check.